lib/engine/game/g_1880/game.rb

```diff
@@ -675,18 +737,47 @@ def operating_order
         end
 
         def after_buying_train(train)
-          return if train.name == @depot.upcoming.first.name
+          return if skip_trigger_sr?(train)
```

Comment:
Nit: More readable if `return unless trigger_sr?(train)`

Reply:
demorgan to the rescue!